android/src/main/java/io/customer/reactnative/sdk/messagingpush/CustomerIORNFirebaseMessagingService.kt

```diff
@@ -0,0 +1,12 @@
+package io.customer.reactnative.sdk.messagingpush
+
+import android.annotation.SuppressLint
+import io.customer.messagingpush.CustomerIOFirebaseMessagingService
+
+/**
+ * Customer.io firebase messaging service class that inherits base messaging service from native
+ * Android SDK to get all basic functionality. This class can be used to include react native
+ * specific features e.g. triggering events to JS/TS, etc.
+ */
+@SuppressLint("MissingFirebaseInstanceTokenRefresh")
+open class CustomerIORNFirebaseMessagingService : CustomerIOFirebaseMessagingService()
```

Comment:
It seems like this file was created just in case we need it in the future. But what if we don't need to use it in the future? Can we create this file/class in the future when there is a problem?

Reply:
Yeah, I don't want to recommend different classes to customers in future as well. Having single dedicated class for react native makes it easier for us to add future changes as well without changing any recommendations. More details on the issue can be read [here](https://github.com/customerio/issues/issues/9232#issuecomment-1440016917).